apps/emqx_authn/src/enhanced_authn/emqx_enhanced_authn_scram_mnesia.erl

```diff
@@ -54,8 +54,9 @@
 
 -define(TAB, ?MODULE).
 -define(AUTHN_QSCHEMA, [
-    {<<"like_username">>, binary},
-    {<<"user_group">>, binary}
+    {<<"like_user_id">>, binary},
+    {<<"user_group">>, binary},
```

Comment:
auth_group ?

Reply:
It is currently only an internal implementation and does not provide this parameter to the user.
If we do need to change it, we can put it after 5.0.